renovate.json

```diff
@@ -16,7 +20,7 @@
     {
       "groupName": "typedoc packages",
       "matchUpdateTypes": ["minor", "major"],
-      "matchPackageNames": ["/^typedoc/"]
+      "matchPackageNames": ["/^typedoc(-|$)/"]
```

Comment:
drive-by - the regex would've also matched `typedocxyz` but we actually just want `typedoc` and packages starting with `typedoc-` (like `typedoc-plugin-missing-exports`)